ci-operator/step-registry/redhat-chaos/cerberus/one-run/redhat-chaos-cerberus-one-run-ref.yaml

```diff
@@ -0,0 +1,21 @@
+ref:
+  as: redhat-chaos-cerberus-one-run
```

Comment:
The documentation for this step states that we target etcd container. If that is true, should we update the step name here to include etcd in it? 
If not, should we be updating documentation for this step?